File: dataset/human/cpp/file_8073.cpp
#include <iostream>
#include <math.h>
#include <algorithm>
#include <string>
using namespace std;

int main() {
	long long int N, b = 0, a[100000 * 2] = {}, ans = 0, sum[100000 * 2 + 1] = {}, dif = 0, mini = 1000000000 * 2 + 1;
	cin >> N;
	for (int i = 0; i < N; i++)
	{
		cin >> a[i];
		sum[i + 1] = sum[i] + a[i];
	}
	for (int i = 0; i < N - 1; i++)
	{
		b = (sum[N] - sum[i + 1]) - sum[i + 1];
		dif = abs(b);
		mini = min(dif, mini);
	}
	cout << mini;
	return 0;
}

File: dataset/human/python/file_8073.py
import std.algorithm, std.container, std.conv, std.math, std.range, std.typecons, std.stdio, std.string;

auto rdsp(){return readln.splitter;}
void pick(R,T)(ref R r,ref T t){t=r.front.to!T;r.popFront;}
void readV(T...)(ref T t){auto r=rdsp;foreach(ref v;t)pick(r,v);}

void main()
{
  int k, s; readV(k, s);

  auto ans = 0;
  foreach (x; 0..k+1)
    foreach (y; 0..k+1) {
      auto z = s-x-y;
      if (z >= 0 && z <= k) ++ans;
    }

  writeln(ans);
}
